scripts/util/get_y_class.py

```diff
@@ -0,0 +1,83 @@
+"""
+(C) Gregory Way 2016
+PanCancer NF1 Classifier
+scripts/util/get_y_class.py
+
+Description:
+Function to determine nearest neighbor of unlabeled data
+
+Usage:
+Import only by build_y.py
+"""
+
+import os
+import pandas as pd
+
+
+def get_y_class(mut, tis, gene=['NF1'], mut_filt=['Silent'],
```

Comment:
Not sure that the complexity (identified by CodeClimate) is the right issue here, but passing a mutable type as a default argument to a python function is dangerous. It will only hit you if you end up calling it multiple times, but it's something to avoid:
Python’s default arguments are evaluated once when the function is defined, not each time the function is called (like it is in say, Ruby). This means that if you use a mutable default argument and mutate it, you will and have mutated that object for all future calls to the function as well.
Python’s default arguments are evaluated once when the function is defined, not each time the function is called (like it is in say, Ruby). This means that if you use a mutable default argument and mutate it, you will and have mutated that object for all future calls to the function as well.

http://docs.python-guide.org/en/latest/writing/gotchas/

Specifically:
>Python’s default arguments are evaluated once when the function is defined, not each time the function is called (like it is in say, Ruby). This means that if you use a mutable default argument and mutate it, you will and have mutated that object for all future calls to the function as well.